Melakukan perubahan pada file

koroyaki opened the pull request.

koroyaki committed:
Melakukan perubahan pada file

BintangRP commented on Hello.txt:
```diff
@@ -1,4 +1,5 @@
 ---
 hal berbeda pada setiap isi dalam hello.txt akan mengakibatkan conflict ketika di merge
+
+- Melakukan ulasan pada setiap perubahan code
```
Perbaiki penggunaan kata yang salah

koroyaki replied:
baik pak

koroyaki replied:
Sudah diperbaiki

koroyaki committed:
Memperbaiki Kata

Memperbaiki penggunaan kata yang salah, serapan bahasa inggris, dan code review

BintangRP commented:
Merge telah dilakukan tetap semangat dalam berkontribusi. You have a great work
